packages/flutter/test/material/bottom_navigation_bar_test.dart

```diff
@@ -704,6 +704,7 @@ void main() {
           data: const MediaQueryData(padding: EdgeInsets.only(bottom: 40.0)),
           child: Scaffold(
             bottomNavigationBar: BottomNavigationBar(
+              selectedFontSize: 8,
```

Comment:
> I'll update this with coverage for a custom `selectedFontSize` as discussed.

We should leave the test that checked the default value as it was and add the custom one, rather than changing the test of the defaults.

Reply:
Gotcha, should be good now 🤞